src/styles/main.scss

```diff
@@ -1,3 +1,181 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document